7b_temp_merge/src/merge_all_temp_dat.R

```diff
@@ -276,27 +276,35 @@ reduce_temp_data <- function(outind, inind) {
 
 }
 
-reduce_reservoir_data <- function(outind, inind) {
+reduce_reservoir_data <- function(outind, drb_ind, nyc_dep_ind) {
   # group by depth category
   # select value closest to noon
-  dat <- readRDS(sc_retrieve(inind)) %>%
+  drb_reservoirs_temps <- readRDS(sc_retrieve(drb_ind)) %>%
     mutate(dateTime_est = as.POSIXct(dateTime, tz = 'America/New_York'),
            time = format(dateTime, '%H:%M'),
            time_hm = lubridate::hm(time),
            hours_minus_noon = time_hm$hour - 12,
            minutes_decimal = time_hm$minute/60,
            hours_diff_noon = abs(hours_minus_noon + minutes_decimal)) %>% arrange(site_id, date, depth_category)
 
-  dat_out <- dat %>%
+  nyc_dep_reservoirs_temps <- readRDS(sc_retrieve(nyc_dep_ind))
+
+  # Selecting unique dates in nyc_dep_reservoirs_temp data.
+  nyc_dep_dates <- unique(nyc_dep_reservoirs_temps$date)
+
+  daily_drb_dat <- drb_reservoirs_temps %>%
+    filter(!date %in% nyc_dep_dates) %>%
     group_by(site_id, date, depth_category) %>%
     slice_min(hours_diff_noon) %>%
     ungroup() %>%
     dplyr::select(site_id, source_id, date, depth, temp)
 
-  saveRDS(dat_out, as_data_file(outind))
-  gd_put(outind)
+  drb_source <- unique(daily_drb_dat$source_id)
 
+  combine_dat <- bind_rows(daily_drb_dat, nyc_dep_reservoirs_temps) %>%
```

Comment:
take a look at the `.id` argument for bind_rows, which should help you create a column that tracks the source of the data (so handles what you're trying to do in line 305)

Reply:
It's okay to leave this as-is, the above comment was to show an alternative method.